frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/place_block_widget.dart';
+
+class TripViewWidget extends StatefulWidget {
+  TripViewWidget({Key key}) : super(key: key);
+  _TripWidgetState createState() => _TripWidgetState();
+}
+
+class _TripWidgetState extends State<TripViewWidget> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+          border: Border.all(
+            color: Colors.black,
+          ),
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      margin: const EdgeInsets.all(15.0),
+      padding: const EdgeInsets.all(10.0),
+      width: MediaQuery.of(context).size.width * .80,
+      child: ExpansionTile(
+        title: Text("Trip Name"),
+        trailing: Text("M T D"),
+        children: [
+          Column(
+            children: [
+              PlaceBlockWidget("Place 1"),
+              PlaceBlockWidget("Place 2"),
+              PlaceBlockWidget("Place 3"),
+              PlaceBlockWidget("Place 4"),
+              PlaceBlockWidget("Place 5"),
+            ],
+          ),
+        ],
+        backgroundColor: Colors.green,
```

Comment:
Nit: trip name is hard to read when this is expanded (light gray on green). Consider getting colors from a theme/color palette or just manually selecting a reasonable one here.

Reply:
```dart
Theme.of(context).accentColor
```
Or any of the other colors [ThemeData](https://api.flutter.dev/flutter/material/ThemeData-class.html) exposes.